precompiles/assets-erc20/src/lib.rs

```diff
@@ -832,9 +835,9 @@ where
 		let mut input = handle.read_input()?;
 		input.expect_arguments(3)?;
 
-		let name: Bytes = input.read::<Bytes>()?.into();
-		let symbol: Bytes = input.read::<Bytes>()?.into();
-		let decimals: u8 = input.read::<u8>()?.into();
+		let name: BoundedBytes<STRING_LIMIT> = input.read()?;
```

Comment:
We can probably use the associated type https://github.com/PureStake/moonbeam/blob/36d572c610be85c322f78e93e70fc7854ba44627/runtime/moonbase/src/asset_config.rs#L58